static/login.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+     <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
+    <title>Login</title>
+    <style>
```

Comment:
Make this a little prettier and consistent with some of the styles from https://github.com/ampproject/amp-by-example/blob/master/templates/css/styles.css